packages/accounts-controller/src/AccountsController.ts

```diff
@@ -868,12 +868,6 @@ export class AccountsController extends BaseController<
                 internalAccounts.accounts,
               ) as InternalAccount[];
 
-              // Get next account name available for this given keyring.
-              const name = this.getNextAvailableAccountName(
```

Comment:
should we also remove the action `AccountsControllerGetNextAvailableAccountNameAction`? And I see the method also being bind

Reply:
@gantunesr I don't think the call would fail since we didn't remove the function itself, just the call to it in the state change handler. 

@ccharly yup, I can make a preview build and test.

Reply:
I also prefer to make these changes in one go